src/content/developers/docs/networking-layer/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Networking layer
+description: An introduction to Ethereum's networking layer.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+Ethereum's networking layer is the stack of protocols that allows nodes to find each other and exchange information. After the merge there will be two pieces of client software (execution clients and consensus clients) each with their own separate networking stack. As well as communicating with other nodes on the Ethereum network, the execution and consensus clients also have to communicate with each other. This page gives an introductory explanation of the protocols that enable this communication to take place.
+
+## Prerequisites {#prerequisites}
+
+Some knowledge of Ethereum [nodes and clients](/src/content/developers/docs/nodes-and-clients/) will be helpful for understanding this page.
+
+## The Execution Layer {#execution-layer}
+
+The execution layer's networking protocols can be divided into two stacks: the first is the discovery stack which is built on top of UDP and allows a new node to find peers to connect to. The second is the [DevP2P](https://github.com/ethereum/devp2p) stack that sits on top of TCP and allows nodes to exchange information. These layers act in parallel, with the discovery layer feeding new network participants into the network, and the DevP2P layer enabling their interactions. DevP2P is itself a whole stack of protocols that are implemented by Ethereum to establish and maintain the peer-to-peer network.
+
+### Discovery {#discovery}
+
+Discovery is the process of finding other nodes in network. This is bootstrapped using a small set of bootnodes that are [hardcoded](https://github.com/ethereum/go-ethereum/blob/master/params/bootnodes.go) into the client. These bootnodes exist to introduce a new node to a set of peers - this is their sole purpose, they do not participate in normal client tasks like syncing the chain, and they are only used the very first time a client is spun up.
```

Comment:
What is a bootnode :-)?

Reply:
ok will add an introductory sentence here.